.travis.yml

```diff
@@ -4,10 +4,18 @@ julia:
   - nightly
 
 env:
- global:
-   - GAPROOT=$TRAVIS_BUILD_DIR/gap
-   - MAKEFLAGS="-j4"
-   - JULIA_TRACK_COVERAGE=yes
+  - GAP=$HOME/.julia/gap.sh
+    GAP_INSTALL_PACKAGES=debug
+    MAKEFLAGS="-j4"
+    JULIA_TRACK_COVERAGE=yes
+    CFLAGS+=--coverage
+    LDFLAGS+=--coverage
```

Comment:
Huh, are you sure += is allowed here?!? I kinda doubt it

Reply:
Looking at the build logs, it definitely doesn't work.

Also, why not keep the common env vars in an "env: global:" ? That's what it's for, no?

Reply:
I will try. It did not work for me the last time I tried. However, it should also work without the `+`.

Reply:
Lets see how it works now.